docs/data/base/components/number-field/number-field.md

```diff
@@ -9,27 +9,51 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/spinbutton/
 
 # Number Field
 
-<p class="description">The Number Field component provides users with a field for number values, with stepper buttons and a scrub area to increment or decrement the value.</p>
+<p class="description">Number Field provides users with a numeric input, with buttons and a scrub area to increment or decrement its value.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
 {{"component": "modules/components/ComponentPageTabs.js"}}
 
-## Introduction
+{{"demo": "UnstyledNumberFieldIntroduction", "defaultCodeOpen": false, "bg": "gradient"}}
```

Comment:
What is the logic behind the style of each CSS technology in this demo?
On the checkbox we have:

- MUI System: blue
- Tailwind CSS: pink
- Plain CSS: grey

Here we have:

- MUI System: grey
- Tailwind CSS: darker grey
- Plain CSS: grey

I personally think having the color switch when you change CSS technology is a great visual indication that something did change.
And having coherent colors throughout the components would be nice.

Reply:
Will redesign the demos in a separate PR.